weave/integrations/openai/gpt_image_utils.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any, Callable, Optional
+
+import rich
+
+from weave.integrations.openai.openai_utils import (
+    create_basic_wrapper_async,
+    create_basic_wrapper_sync,
+    maybe_unwrap_api_response
+)
+from weave.trace.autopatch import OpSettings
+from weave.trace.weave_client import Call
+
+
+def openai_image_postprocess_inputs(inputs: dict[str, Any]) -> dict[str, Any]:
+    """
+    Postprocess inputs of the trace for the OpenAI Image API to be used in
+    the trace visualization in the Weave UI.
+    """
+    return inputs
+
+
+def openai_image_postprocess_outputs(outputs: dict[str, Any]) -> dict[str, Any]:
+    """
+    Postprocess outputs of the trace for the OpenAI Image API.
+    Creates a 250x250 thumbnail for logging to Weave while preserving original output.
+    """
+    from io import BytesIO
+
+    from PIL import Image  # type: ignore
+
+    # Unwrap API response if needed
+    outputs = maybe_unwrap_api_response(outputs)
+    
+    # Create a copy of outputs for Weave logging
+    weave_outputs = outputs.copy() if isinstance(outputs, dict) else outputs
+    
+    if isinstance(outputs, dict) and "data" in outputs:
+        for i, image_data in enumerate(outputs["data"]):
+            if "b64_json" in image_data:
+                import base64
+                try:
+                    image_bytes = base64.b64decode(image_data["b64_json"])
+                    pil_image = Image.open(BytesIO(image_bytes))
+                    # Create a thumbnail version for Weave logging
+                    thumbnail = pil_image.copy()
+                    thumbnail_size = (250, 250)
```

Comment:
Why are we using (250,250)? 

Also if we are logging a thumbnail, we should add a `original_size` or `original_shape` key as well.